src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_utils.py

```diff
@@ -449,3 +449,17 @@ def interrupt_main_thread(main_thread):
         except:
             pydev_log.exception('Error on interrupt main thread fallback.')
 
+
+def current_thread():
+    try:
+        impl = threading.current_thread
+    except AttributeError:
+        impl = threading.currentThread
+    return impl()
+
+
+def get_thread_name(t):
```

Comment:
`threading.Thread.name` is already available in Python 2.6, so, I don't think we actually need to do the try..except.

Maybe we can just inline this method and just use `t.name` where needed...